### ISSUE TITLE
Raw markdown and html-tags visible in vcd_vapp_vm resource documentation

### ISSUE THREAD

**Issue by anttiah** [vmware/terraform-provider-vcd#840]
There appears to be a missing closing triple backtick somewhere in vcd_vapp_vm documentaion, causing code block formatting to spill over bottom half of the page. It makes hard to follow the documentation.

### Affected Resource(s)
- vcd_vapp_vm documentation
 https://github.com/vmware/terraform-provider-vcd/blob/main/website/docs/r/vapp_vm.html.markdown

### Expected Behavior
Page is properly formatted.

### Actual Behavior
Page shows raw markdown and html-tags.

### Steps to Reproduce
Browse to either of the pages
- https://github.com/vmware/terraform-provider-vcd/blob/main/website/docs/r/vapp_vm.html.markdown
- https://registry.terraform.io/providers/vmware/vcd/latest/docs/resources/vapp_vm

and scroll to about half way down the page. Make note how formatting changes from normal web page style to raw markdown and html-tags.

**Comment by vbauzysvmware** [comment]
This https://github.com/vmware/terraform-provider-vcd/pull/828 one will fix this.

**Comment by Didainius** [comment]
Merged to master.
